### PR TITLE
[8.0][WIP] prima nota cassa

### PR DESCRIPTION
**Pull request by dcorio**
v8 porting of l10n_it_prima_nota_cassa
mako > qweb

### CODE REVIEW

**Comment by eLBati** [comment]
Thanks.
Please see travis
```
./l10n_it_prima_nota_cassa/wizard/wizard_print_prima_nota_cassa.py:44:80: E501 line too long (118 > 79 characters)
./l10n_it_prima_nota_cassa/report_primanotacassa.py:9:80: E501 line too long (80 > 79 characters)
./l10n_it_prima_nota_cassa/report_primanotacassa.py:26:80: E501 line too long (83 > 79 characters)
./l10n_it_prima_nota_cassa/report_primanotacassa.py:35:80: E501 line too long (114 > 79 characters)
./l10n_it_prima_nota_cassa/report_primanotacassa.py:36:9: F841 local variable 'ctx2' is assigned to but never used
./l10n_it_prima_nota_cassa/report_primanotacassa.py:52:80: E501 line too long (89 > 79 characters)
./l10n_it_prima_nota_cassa/report_primanotacassa.py:86:9: F841 local variable 'currency_obj' is assigned to but never used
./l10n_it_prima_nota_cassa/report_primanotacassa.py:93:80: E501 line too long (110 > 79 characters)
./l10n_it_prima_nota_cassa/report_primanotacassa.py:95:80: E501 line too long (122 > 79 characters)
./l10n_it_prima_nota_cassa/report_primanotacassa.py:98:9: F841 local variable 'currency' is assigned to but never used
./l10n_it_prima_nota_cassa/report_primanotacassa.py:155:80: E501 line too long (94 > 79 characters)
```

File: l10n_it_prima_nota_cassa/__openerp__.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # -*- coding: utf-8 -*-
 ##############################################################################
-#    
+#
 #    Copyright (C) 2011-2013 Associazione OpenERP Italia
-#    (<http://www.openerp-italia.org>). 
+#    (<http://www.openerp-italia.org>).
```
Comment (eLBati) [inline l10n_it_prima_nota_cassa/__openerp__.py]: Please update URL

File: l10n_it_prima_nota_cassa/__openerp__.py [hunk 2]
```diff
@@ -22,18 +22,19 @@
     'name': 'Italian Localisation - Prima Nota Cassa',
     'version': '0.1',
     'category': 'Localisation/Italy',
-    'description': """Accounting reports - Prima Nota Cassa - Webkit""",
+    'description': """Accounting reports - Prima Nota Cassa""",
     'author': "OpenERP Italian Community,Odoo Community Association (OCA)",
     'website': 'http://www.openerp-italia.org',
```
Comment (eLBati) [inline l10n_it_prima_nota_cassa/__openerp__.py]: Please update URL

**Comment by eLBati** [comment]
Please remove ```AUTHORS.txt``` file

File: l10n_it_prima_nota_cassa/wizard/wizard_print_prima_nota_cassa.py [hunk 3]
```diff
@@ -21,34 +21,43 @@
...
+        return {
+            'type': 'ir.actions.report.xml',
+            'report_name': 'l10n_it_prima_nota_cassa.report_primanotacassa',
+            'datas': data
+        }
+
+    initial_balance = fields.Boolean(
+        'Include initial balances',
+        help="""It adds initial balance row on report which display previous
+sum amount of debit/credit/balance""")
```
Comment (eLBati) [inline l10n_it_prima_nota_cassa/wizard/wizard_print_prima_nota_cassa.py]: You can write
```
        help="It adds initial balance row on report which display previous "
             "sum amount of debit/credit/balance")
```

**Comment by eLBati** [comment]
Maybe a layout issue: try to go to
http://3114295-132-78a843.runbot2.odoo-community.org/web?db=3114295-132-78a843-base
and print the report with 'all entries', the 4th record splits in 2 lines

**Comment by maxhome1** [comment]
In the press misses "reference" (Ref) field. 
Wouldn't it be more readable for horizontal or with a lower character?

**Comment by dcorio** [comment]
@eLBati thank you very much for the review :)

**Comment by eLBati** [comment]
it needs rebase

**Comment by eLBati** [comment]
No activity for more than 6 months, closing. Feel free to reopen

**Pull request closed without merging**